bulletin/index.html

```diff
@@ -203,6 +203,53 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
 	  </div>
 	</section>
 
+	<section class="mention">
+		<div class="mention-container mention-container__wrapper">
+			<div class="mention-heading">
+				<div> 
+					<h2 class="mention-heading__title">Comments & Acknowledgments</h2>
+				</div >
+				<div>
+					<p class="mention-heading__description">We recognize the contributions and heartfelt comments of the people, companies, and communities that characterize MACILE. They are invaluable. Without them, improving the  program and meeting its objectives would be difficult.</p>
+				</div>
+			</div>
+			<div class="mention-section">
+				<h3 class="mention-section__subtitle">From Our Students</h3>
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item">This program helped me fall in love with mathematics and language. It gave me new dreams and hopes. I have no words to express my gratitude to MACILE for bringing us what I like best, science, and for instilling in me a passion for learning.</blockquote> 
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item">This summer, I learned to read better, how to express my thoughts, and how to analyze problems.</blockquote>
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item"> I learned many examples of good behavior and about capitalization.</blockquote>	
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item">I learned a lot about bridges, solar energy, mathematics, and other things. I also learned that I should not speak with a loud voice and not to disturb others. I also learned that working on a team is extremely important to achieve my goals.
+				</blockquote>
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item"> I learned that I should ask questions when in doubt. If I remain silent, I will never learn what I need to learn. I also learned some mathematics and Spanish. I did not know how to write an essay. I used to make a textual copy of the material.</blockquote>
+			</div>
+			<div class="mention-section">
+				<h3 class="mention-section__subtitle">Acknowledgments</h3>
+			</div>
```

Comment:
Improving the structure a bit more. Each of the `sections`, student comments and acknowledgements, could be wrapped in a `section` element. The headers (h3) that introduce each section can then use `header` . `mention-section` more aptly names the section not a header. 

Also by wrapping the content in a `section` it helps screen readers know that all the content within the section tags relates to its header and it in code it is much easier to distinguish the blocks of content.

Example of the change in code

```
<section>
  <header><h3>Acknowledgements</h3><header>
  <div>....</div>
</section>
```

Reply:
Nice improvement. Please review the changes. Thanks.